package.json

```diff
@@ -36,8 +36,11 @@
     "css-loader": "^0.21.0",
     "express": "^4.13.3",
     "extract-text-webpack-plugin": "^0.9.0",
+    "node-sass": "^3.4.1",
+    "normalize.css": "^3.0.3",
```

Comment:
Yes the initial build fails without `node-sass`, `normalize.css` for anyone using npm 3.0+, as peerDependencies will not be installed automatically. npm will warn the user that there are unmet peerDependencies, which is fine for anyone using `react-toolbox` in their own project. I think it would be useful for the example to include the peerDependencies as a devDependencies so people can just clone, install, and run.

Here the output after running `npm i` (npm v3.3.6)

```sh
npm WARN EPEERINVALID sass-loader@3.1.1 requires a peer of node-sass@^3.3.3 but none was installed.
npm WARN EPEERINVALID react-toolbox@0.11.3 requires a peer of react-addons-css-transition-group@^0.14.0 but none was installed.
npm WARN EPEERINVALID react-toolbox@0.11.3 requires a peer of normalize.css@^3.0.3 but none was installed.
```

On Ubuntu 15.04, the build failed without `react-addons-css-transition-group`, but did not on OSX. Currently, I'm trying to get to the bottom of that.